content/blog/2019/egress-traffic-control-in-istio-part-3/index.md

```diff
@@ -70,67 +69,72 @@ are not transparent and not Kubernetes-aware.
-The latency of access to external services could be already high, so adding the overhead
-of two or three proxies inside the cluster could be not very significant.
-After all, in the microservice architecture you can have chains of dozens of calls between microservices, so adding an
-additional hop with one or two proxies in the egress gateway should not have a large impact.
+Let me share my thoughts on the performance overhead that controlling egress traffic using Istio adds:
+Accessing external services already could have high latency and the overhead added
+because of two or three proxies inside the cluster could likely not be very significant by comparison.
+After all, applications with a microservice architecture can have chains of dozens of calls between microservices.
+Therefore, an additional hop with one or two proxies in the egress gateway should not have a large impact.
+
+Moreover, we continue to work towards reducing Istio's performance overhead.
+Possible optimizations include:
 
-Moreover, we are working to reduce performance overhead of Istio.
-Possible optimizations are to extend Envoy to handle wildcard domains so there will be no need for the
-third proxy; or to use mutual TLS for authentication only, without encrypting the TLS traffic (since it is already
-encrypted).
+- Extending Envoy to handle wildcard domains: This would eliminate the need for a third proxy between
+  the application and the external services for that use case.
+- Using mutual TLS for authentication only without encrypting the TLS traffic, since the traffic is already
+  encrypted.
 
 ## Summary
 
 I hope that after reading this series you are convinced that controlling egress traffic is very important for the
 security of your cluster.
-I also hope that I managed to convince you that Istio can serve as an effective tool for controlling egress traffic
+Hopefully, I also managed to convince you that Istio is an effective tool to control egress traffic
 securely, and that Istio has multiple advantages over the alternative solutions.
-In my opinion, you can even choose secure control of egress traffic as the first use case for applying Istio to your
-cluster.
-Istio will already be beneficial for you, even before you start using all other features, such as
-traffic management, security, policies and telemetry, applied to traffic between microservices inside the cluster.
-You should pay attention, however, to performance overhead of Istio egress traffic control and measure it for your use
-cases.
+In my opinion, secure control of egress traffic is a great choice if you are looking for your first Istio use case.
+In this case, Istio already provides you some benefits even before you start using all other Istio features:
+[traffic management](/docs/tasks/traffic-management/), [security](/docs/tasks/security/),
+[policies](/docs/tasks/policy-enforcement/) and [telemetry](/docs/tasks/telemetry/), applied to traffic between microservices inside the cluster.
```

Comment:
@vadimeisenbergibm What is the final message you want to give readers after they have followed you through all three parts? The message on this finishing paragraph is a bit muddled. Try to picture someone who followed the entire journey. They learned what controlling egress traffic securely was all about, the different attacks that make it important, and the different ways they can protect against said attacks. After all of that, you reached the conclusion that Istio's secure control of egress traffic is the way to go and explained why. 
IMO, this paragraph should communicate three things to your readers:
1. What's next? Assume readers have followed along and secured their egress traffic, what specific actions would you recommend they take next?
2. Where can I learn more? I think this is what you were trying to do by listing the features. However, those still feel disconnected from the topic at hand. What tasks do you recommend they complete next? What reference materials will allow your readers to do more around traffic management?
3. Summarize the value that Istio gives them and that they cannot get any other way.
Some of this things are alluded to so you might only need to rearrange and reword a little.

Reply:
Addressed in https://github.com/istio/istio.io/pull/4637/commits/f2a4c183b96d82deed21f3576e46b220063c9deb and https://github.com/istio/istio.io/pull/4637/commits/667c470c82b0a25c882b12c5b40d02b11647a7b6